python/src/bxsolana_trader_proto/api/__init__.py

```diff
@@ -963,6 +963,11 @@ class GetBlockStreamResponse(betterproto.Message):
     timestamp: datetime = betterproto.message_field(2)
 
 
+@dataclass(eq=False, repr=False)
```

Comment:
Should pyproject.toml's version field be updated at this point?

Reply:
yeah

Reply:
updated